cupy/cuda/nccl.pxd

```diff
@@ -14,3 +14,18 @@ cpdef enum:
     NCCL_DOUBLE = 4
     NCCL_INT64 = 5
     NCCL_UINT64 = 6
+
+    NCCL2_INT8 = 0
```

Comment:
Certainly, we can take that approach.

However it may break sort of compatibility. The enum `NCCL_FLOAT` of nccl version 1 is used in chainer, specifically at [multiprocess_parallel_updater.py](https://github.com/chainer/chainer/blob/master/chainer/training/updaters/multiprocess_parallel_updater.py). So, if we take that route, users have to upgrade chainer and cupy to appropriate versions respectively at the same time.